Cargo.toml

```diff
@@ -34,7 +34,9 @@ p384 = { version = "0.8", default-features = false, features = ["ecdsa"] }
 pbkdf2 = { version = "0.9", optional = true, default-features = false }
 serde = { version = "1", features = ["serde_derive"] }
 serde_json = { version = "1", optional = true }
+rand = "0.8"
 rand_core = { version = "0.6", features = ["std"] }
+rsa = "0.5"
```

Comment:
Since this is only used for testing, it should be added to `[dev-dependencies]`.